include/sir/config.h

```diff
@@ -373,8 +373,14 @@
  * data in any color mode (the largest possible sequence, which is:
  * `\x1b[a;fb;m;rrr;ggg;bbb;fb;m;rrr;ggg;bbbm`) plus a null terminator.
  */
-# if !defined(SIR_MAXSTYLE)
-#  define SIR_MAXSTYLE 43
+# if !defined(SIR_NO_TEXT_STYLING)
```

Comment:
if `SIR_EOL_CRLF` is defined we need to add 1 to `SIR_MAXSTYLE`

Reply:
Yep.